src/olympia/api/validators.py

```diff
@@ -0,0 +1,17 @@
+import unicodedata
+
+from django.utils.translation import ugettext
+
+from rest_framework import serializers
+
+
+class OneOrMorePrintableCharacterValidator:
+    def __init__(self, unicode_categories=None):
+        self.unicode_categories = unicode_categories or ('L', 'N', 'P', 'S')
```

Comment:
Can you add a comment linking to http://www.unicode.org/reports/tr44/tr44-6.html#Property_Values and explaining which categories those are, for future reference ?

Reply:
https://github.com/mozilla/addons-server/pull/10324/commits/fbb73165a83cf05a76479bd5373bed318d79caf7